{
	"cve": {
		"CVE_data_meta": {
			"ASSIGNER": "cve@mitre.org",
			"ID": "CVE-2019-6446"
		},
		"data_format": "MITRE",
		"data_type": "CVE",
		"data_version": "4.0",
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "An issue was discovered in NumPy 1.16.0 and earlier. It uses the pickle Python module unsafely, which allows remote attackers to execute arbitrary code via a crafted serialized object, as demonstrated by a numpy.load call. NOTE: third parties dispute this issue because it is  a behavior that might have legitimate applications in (for example)  loading serialized Python object arrays from trusted and authenticated  sources"
				}
			]
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": [
						{
							"lang": "en",
							"value": "CWE-502"
						}
					]
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"name": "openSUSE-SU-2019:2227",
					"url": "http://lists.opensuse.org/opensuse-security-announce/2019-09/msg00091.html"
				},
				{
					"name": "openSUSE-SU-2019:2225",
					"url": "http://lists.opensuse.org/opensuse-security-announce/2019-09/msg00092.html"
				},
				{
					"name": "openSUSE-SU-2019:2259",
					"url": "http://lists.opensuse.org/opensuse-security-announce/2019-10/msg00015.html"
				},
				{
					"name": "106670",
					"tags": [
						"Third Party Advisory",
						"VDB Entry"
					],
					"url": "http://www.securityfocus.com/bid/106670"
				},
				{
					"name": "RHSA-2019:3335",
					"url": "https://access.redhat.com/errata/RHSA-2019:3335"
				},
				{
					"name": "RHSA-2019:3704",
					"url": "https://access.redhat.com/errata/RHSA-2019:3704"
				},
				{
					"name": "https://bugzilla.suse.com/show_bug.cgi?id=1122208",
					"tags": [
						"Exploit",
						"Issue Tracking",
						"Third Party Advisory"
					],
					"url": "https://bugzilla.suse.com/show_bug.cgi?id=1122208"
				},
				{
					"name": "https://github.com/numpy/numpy/issues/12759",
					"tags": [
						"Exploit",
						"Issue Tracking",
						"Third Party Advisory"
					],
					"url": "https://github.com/numpy/numpy/issues/12759"
				},
				{
					"name": "FEDORA-2019-1dfe95a864",
					"url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/7ZZAYIQNUUYXGMKHSPEEXS4TRYFOUYE4/"
				},
				{
					"name": "openSUSE-SU-2019:2227",
					"url": "http://lists.opensuse.org/opensuse-security-announce/2019-09/msg00091.html"
				},
				{
					"name": "FEDORA-2019-1dfe95a864",
					"url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/7ZZAYIQNUUYXGMKHSPEEXS4TRYFOUYE4/"
				},
				{
					"name": "https://github.com/numpy/numpy/issues/12759",
					"tags": [
						"Exploit",
						"Issue Tracking",
						"Third Party Advisory"
					],
					"url": "https://github.com/numpy/numpy/issues/12759"
				},
				{
					"name": "https://bugzilla.suse.com/show_bug.cgi?id=1122208",
					"tags": [
						"Exploit",
						"Issue Tracking",
						"Third Party Advisory"
					],
					"url": "https://bugzilla.suse.com/show_bug.cgi?id=1122208"
				},
				{
					"name": "RHSA-2019:3704",
					"url": "https://access.redhat.com/errata/RHSA-2019:3704"
				},
				{
					"name": "RHSA-2019:3335",
					"url": "https://access.redhat.com/errata/RHSA-2019:3335"
				},
				{
					"name": "106670",
					"tags": [
						"Third Party Advisory",
						"VDB Entry"
					],
					"url": "http://www.securityfocus.com/bid/106670"
				},
				{
					"name": "openSUSE-SU-2019:2259",
					"url": "http://lists.opensuse.org/opensuse-security-announce/2019-10/msg00015.html"
				},
				{
					"name": "openSUSE-SU-2019:2225",
					"url": "http://lists.opensuse.org/opensuse-security-announce/2019-09/msg00092.html"
				}
			]
		}
	},
	"impact": {
		"baseMetricV2": {
			"cvssV2": {
				"accessComplexity": "LOW",
				"accessVector": "NETWORK",
				"authentication": "NONE",
				"availabilityImpact": "PARTIAL",
				"baseScore": 7.5,
				"confidentialityImpact": "PARTIAL",
				"integrityImpact": "PARTIAL",
				"vectorString": "AV:N/AC:L/Au:N/C:P/I:P/A:P",
				"version": "2.0"
			},
			"exploitabilityScore": 10,
			"impactScore": 6.4,
			"severity": "HIGH"
		},
		"baseMetricV3": {
			"cvssV3": {
				"attackComplexity": "LOW",
				"attackVector": "NETWORK",
				"availabilityImpact": "HIGH",
				"baseScore": 9.8,
				"baseSeverity": "CRITICAL",
				"confidentialityImpact": "HIGH",
				"integrityImpact": "HIGH",
				"privilegesRequired": "NONE",
				"scope": "UNCHANGED",
				"userInteraction": "NONE",
				"vectorString": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
				"version": "3.0"
			},
			"exploitabilityScore": 3.9,
			"impactScore": 5.9
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": [
			{
				"cpe_match": [
					{
						"cpe23Uri": "cpe:2.3:a:numpy:numpy:*:*:*:*:*:*:*:*",
						"versionEndIncluding": "1.16.0",
						"vulnerable": true
					}
				],
				"operator": "OR"
			},
			{
				"cpe_match": [
					{
						"cpe23Uri": "cpe:2.3:o:fedoraproject:fedora:30:*:*:*:*:*:*:*",
						"vulnerable": true
					}
				],
				"operator": "OR"
			}
		]
	},
	"lastModifiedDate": "2024-11-21T04:46Z",
	"publishedDate": "2019-01-16T05:29Z"
}
